mm-go-irckit/userbridge.go

```diff
@@ -1111,3 +1105,53 @@ func (u *User) handleMessageThreadContext(channelID, messageID, parentID, event,
 
 	return newText, prefix, suffix, showContext, maxlen
 }
+
+//nolint:gocyclo
+func (u *User) formatCodeBlockText(text string, prefix string, codeBlockBackTick bool, codeBlockTilde bool, lexer string) (string, bool, bool, string) {
```

Comment:
you only use the first string in the return, so maybe replace it to 

```suggestion
func (u *User) formatCodeBlockText(text string, prefix string, codeBlockBackTick bool, codeBlockTilde bool, lexer string) (string) {
```

Reply:
Actually, these are used to preserve if/when code blocks start and end.